test/127489_engine_params.js

```diff
@@ -97,24 +113,5 @@ describe('127489 engine parameters Starboard', function () {
       'notifications.propulsion.starboard.maintenanceNeeded.value.state',
       'alarm')
     tree.should.be.validSignalKVesselIgnoringIdentity
-
```

Comment:
Why remove this test?

Reply:
I thought I copied a bit of code twice, so removed the duplicate. Didn't realise it was a discrete test ;)